pkg/kapp/resourcesmisc/custom_waiting_resource.go

```diff
@@ -151,6 +152,9 @@ func (s CustomWaitingResource) IsDoneApplying() DoneApplyState {
 						cond.Type, condMatcher.Status, cond.Reason)
 					continue
 				}
+				if condMatcher.ShowActualMessage {
+					actualMsg = cond.Message
```

Comment:
isn't this message will be overridden in the loop?

Reply:
No, because this code will get executed only when `cond.Type == condMatcher.Type && cond.Status == condMatcher.Status`. This will happen only once.